web/apps/labelstudio/src/pages/Organization/PeoplePage/InviteLink.tsx

```diff
@@ -0,0 +1,110 @@
+import { Description } from "apps/labelstudio/src/components/Description/Description";
+import { Block } from "apps/labelstudio/src/components/Menu/MenuContext";
+import { Input } from "../../../components/Form";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { Space } from "@humansignal/ui";
+import { Button } from "@humansignal/shad/components/ui/button";
+import { API } from "apps/labelstudio/src/providers/ApiProvider";
+import { atomWithQuery } from "jotai-tanstack-query";
+import { useAtomValue } from "jotai";
+import { Modal } from "apps/labelstudio/src/components/Modal/ModalPopup";
+
+const linkAtom = atomWithQuery(() => ({
```

Comment:
Just a note and doesn't have to be done in this PR. As an FE group we will want to review atom organization and probably lift these things up to another file/directory for easier understanding as to usage patterns. Similarily I made a call out to any inline `useQuery` calls should be pulled out of any component file and created as a specific hook which can then be imported and reused elsewhere. No action required here, just making a note 🙂 
```
const linkAtom = atomWithQuery(() => ({
```

Reply:
i believe that there will be cases when we want to inline queries, but each should be justified separately